.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+per-file-ignores =
+    tests/test_scalars.py: W503
```

Comment:
I tested your PR with this removed, and it passed, so can you remove this file?

Reply:
Removed